src/main/java/com/epam/service/TourOfferService.java

```diff
@@ -27,29 +27,29 @@ public TourOfferService(TourOfferDAO tourOfferDAO){
     }
 
     public int deleteTour(Integer tourId){
-        if(tourId!=null&&tourId!=0)
-            return tourOfferDAO.deleteTour(tourId);
-        else {
+        if (tourId == null || tourId == 0) {
```

Comment:
What's wrong with `0`?

Reply:
I believe we dont want to save 0 id... no?

Reply:
I think `id ` is auto-generated by H2